src/python/pants/backend/python/rules/run_setup_py_test.py

```diff
@@ -485,7 +501,7 @@ def test_owned_dependencies(self) -> None:
                     sources=[],
                     dependencies=[':bar-resources', 'src/python/foo/bar/baz:baz2'],
                 )
-                resources(name='bar-resources')
+                resources(name='bar-resources', sources=[])
```

Comment:
`resources` must define the `sources` field. It doesn't make sense to have a `resources` target without sources in it.